src/main/telemetry/hott.c

```diff
@@ -209,12 +211,34 @@ void hottPrepareGPSResponse(HOTT_GPS_MSG_t *hottGPSMessage)
 }
 #endif
 
+static bool time_for_Voice_Alarm(void)
```

Comment:
methods that return a boolean ideally should start with 'is' or 'should' so that when used in an if statement it reads better.

perhaps in this case:

`if (shouldTriggerVoiceAlarmNow())`